ios/MaasAccount/Sources/MaasAccount/Requirements/Compositions/RequirementsListView.swift

```diff
@@ -0,0 +1,89 @@
+import SwiftUI
+import MaasTheme
+import MaasComponents
+
+public struct RequirementsListView: View, Swappable {
+
+    // NOTE: Should we use environment or just pass values via inits
```

Comment:
I'd be in favor of defaulting to passing via inits unless we have a very good reason to use environment.